pyo3-macros-backend/src/method.rs

```diff
@@ -228,6 +228,18 @@ impl FnType {
         }
     }
 
+    pub fn signature_attribute_allowed(&self) -> bool {
```

Comment:
I think it'd be helpful for future us to add a comment explaining why (getter/setter/classattr have fixed signatures with a single arg where `Option<>` has a specific meaning)

Reply:
Good idea, added one, WDYT?

Reply:
LGTM. Thanks